src/mailchimp/mailchimp_manager.py

```diff
@@ -60,8 +72,14 @@ def ping(self) -> Response:
         Raises:
             TODO: Detail potential errors
         """
-        raise NotImplementedError
-    
+        try:
+                client = MailchimpMarketing.Client()
+                client.set_config(self.keys)
+                response = client.ping.get()
+                print(response)
+        except ApiClientError as error:
+                print("Error: {}".format(error.text))
```

Comment:
Raise errors, don't print them. How errors are handled is not the responsibility of the mailchimp manager.